cleanlab/filter.py

```diff
@@ -673,18 +675,17 @@ def _reduce_prune_counts(prune_count_matrix, frac_noise=1.0):
 
     Parameters
     ----------
-
     prune_count_matrix : np.array of shape (K, K), K = number of classes
         A counts of mislabeled examples in every class. For this function, it
         does not matter what the rows or columns are, but the diagonal terms
         reflect the number of correctly labeled examples.
 
     frac_noise : float
-        When frac_noise = 1.0, return all estimated noise indices.
-        Value in range (0, 1] that determines the fraction of noisy example
-        indices to return based on the following formula for example class k.
-        frac_noise * number_of_mislabeled_examples_in_class_k, or
-        frac_noise * inverse_noise_rate_class_k * num_examples_s_equal_k."""
+      Used to only return the "top" ``frac_noise * num_label_issues``. The choice of which "top"
+      label issues to return is dependent on the `filter_by` method used. It works by reducing the
+      size of the off-diagonals of the `prune_count_matrix` of given labels and true labels
+      proportionally by `frac_noise` prior to estimating label issues with each method.
```

Comment:
Would keep this sentence from the other docstring which seems important:

"When frac_noise=1.0, return all "confident" estimated noise indices (recommended)."